code/modules/events/grid_check.dm

```diff
@@ -14,63 +21,50 @@
 		SEND_SOUND(M, S)
 
 /datum/event/grid_check/announce()
-	GLOB.event_announcement.Announce("Abnormal activity detected in [station_name()]'s powernet. As a precautionary measure, the station's power will be shut off for an indeterminate duration.", "Automated Grid Check", new_sound = 'sound/AI/poweroff.ogg')
+	GLOB.event_announcement.Announce("Abnormal activity detected in [station_name()]'s powernet. As a precautionary measure, the station's power will be shut off for an indeterminate duration.", "Systems Power Failure", new_sound = 'sound/AI/poweroff.ogg')
 
 /datum/event/grid_check/end()
-	power_restore()
-
-/proc/power_failure(var/announce = 1)
-	if(announce)
-		GLOB.event_announcement.Announce("Abnormal activity detected in [station_name()]'s powernet. As a precautionary measure, the station's power will be shut off for an indeterminate duration.", "Critical Power Failure", new_sound = 'sound/AI/poweroff.ogg')
+	return TRUE
 
-	var/list/skipped_areas = list(/area/turret_protected/ai)
-	var/list/skipped_areas_apc = list(/area/engine/engineering)
+/proc/power_failure(announce=TRUE)
+	var/list/skipped_areas_apc = list(
+		/area/engine/engineering,
+		/area/turret_protected/ai)
 
-	for(var/obj/machinery/power/smes/S in GLOB.machines)
-		var/area/current_area = get_area(S)
-		if((current_area.type in skipped_areas) || !is_station_level(S.z))
-			continue
-		S.last_charge			= S.charge
-		S.last_output_attempt	= S.output_attempt
-		S.last_input_attempt 	= S.input_attempt
-		S.charge = 0
-		S.inputting(0)
-		S.outputting(0)
-		S.update_icon()
-		S.power_change()
+	if(announce)
+		GLOB.event_announcement.Announce("Abnormal activity detected in [station_name()]'s powernet. As a precautionary measure, the station's power will be shut off for an indeterminate duration.", "Systems Power Failure", new_sound = 'sound/AI/poweroff.ogg')
```

Comment:
Use:
Overload detected in [station_name()]'s powernet. Engineering, please repair damaged APCs.